modules/core/test/core.bench.js

```diff
@@ -1,26 +1,29 @@
-const LENGTH = 10000;
+const LENGTH = 1e3;
 const ARRAY = new Array(LENGTH).fill(0);
 
 // TODO - Async benchmarks pending probe.gl update
-// async function *asyncIterator(array) {
-//   for (const element of array) {
-//     yield Promise.resolve(element);
-//   }
-// }
+async function* asyncIterator(array) {
```

Comment:
why do we need this generators here?

Reply:
This benchmark compares the speed of non-async iteration with async iteration. 

Async iteration is significantly slower.